addons/mail/static/src/components/composer/composer.scss

```diff
@@ -131,8 +131,9 @@
     margin: 0 2px;
 
     &:not(.o-mobile) {
-        margin-top: 2px;
-        margin-bottom: 2px;
+        margin-top: map-get($spacers, 1);
+        margin-bottom: map-get($spacers, 1);
+        margin-inline-end: map-get($spacers, 2);
```

Comment:
What was the issue resolved with this additional `margin-inline-end` ? Shouldn't the above `margin: 0 2px;` be changed to `margin: 0 map-get($spacers, 2);` if we just need extra horizontal margin? Is that really a non-mobile specific issue?

Reply:
It's linked to the remove rule below which was very complex for what it does and was messing the style because of the popover of emoji button (problem with the first last-child selector if I remember well)

```scss
    &:not(:last-child) .o_Composer_toolButton:last-child {	
        margin-right: 5px;	
    }
```

Reply:
Ok but then it can be removed now too?

Can you spot anything wrong if you remove this extra margin?

It seems fine to me without it.

Reply:
With

![image](https://user-images.githubusercontent.com/48204996/85720781-09711180-b6f1-11ea-8599-c9903a2eb03f.png)

Without

![image](https://user-images.githubusercontent.com/48204996/85720905-26a5e000-b6f1-11ea-8cae-15109d54b78d.png)

a little bit weird no ?

Reply:
Yep, I pushed adjustments so that it's not so close to the left/right while keeping a reduced margin between the 2 buttons.